composer/datasets/dataset_hparams.py

```diff
@@ -120,15 +120,12 @@ class DatasetHparams(hp.Hparams, abc.ABC):
         shuffle (bool): Whether to shuffle the dataset. Default: ``True``.
     """
 
-    is_train: bool = hp.optional('Whether to load the training data (the default) or validation data.', default=True)
     drop_last: bool = hp.optional(textwrap.dedent("""\
         If the number of samples is not divisible by the batch size,
         whether to drop the last batch (the default) or pad the last batch with zeros."""),
                                   default=True)
     shuffle: bool = hp.optional('Whether to shuffle the dataset for each epoch. Defaults to True.', default=True)
 
-    datadir: Optional[str] = hp.optional('The path to the data directory', default=None)
-
     @abc.abstractmethod
```

Comment:
Cam upi also remove the "datadir" and "is_train" arguments from the docstring above?

Reply:
yes, good catch